src/services/deduplicationService.ts

```diff
@@ -0,0 +1,133 @@
+import { EnteFile } from 'types/file';
+import { getEndpoint } from 'utils/common/apiUtil';
+import { getToken } from 'utils/common/key';
+import { logError } from 'utils/sentry';
+import { getLocalFiles } from './fileService';
+import HTTPService from './HTTPService';
+
+const ENDPOINT = getEndpoint();
+
+interface DuplicatesResponse {
+    duplicates: Array<{
+        fileIDs: number[];
+        size: number;
+    }>;
+}
+
+interface DuplicateFiles {
+    files: EnteFile[];
+    size: number;
+}
+
+class DeduplicationService {
```

Comment:
doesn't make sense to make it a class if we don't have any local state, we can just make it a collection of exported functions like `fileService`